modules/installation-and-upgrade/pages/containerized-proxy-deployment.adoc

```diff
@@ -0,0 +1,92 @@
+[[containerized-proxy-deployment]]
+= Containerized proxy deployment using internal registry
+
+It is possible to deploy containerized images in an environment without an internet connection.
+In such case, the images can be copied from {suse} registry to an internal registry, or saved to a [literal]``tar`` file.
+
+
+[[image.from.suse.to.internal.registry]]
+== Image copying from {suse} registry to internal registry 
+
+
+This example illustrates deployment of {salt} proxies only.
+
+.Procedure: Deploying {salt} Proxy from an internal image registry
+[role=procedure]
+
+. Install [literal]``skopeo`` on a machine with access to [literal]``registry.suse.com``:
++
+----
+zypper in skopeo (can be {productname} server)
+----
++
+. Copy images between registries:
++
+----
+for image in httpd salt-broker squid ssh tftpd; do
+    skopeo copy docker://registry.suse.com/suse/manager/4.3/proxy-$image:latest docker://(your_server)/registry.suse.com/suse/manager/4.3/proxy-$image
+done
+skopeo copy docker://k8s.gcr.io/pause:latest docker://(your_server)/k8s.gcr.io/pause:latest
+----
++
+. If the registry is unsecured, i.e. not configured with SSL, add the regitry domain to the section [literal]``registries.insecure`` on the containerized proxy virtual machine by editing:
++
+----
+/etc/containers/registries.conf
+----
++
+[NOTE]
+====
+If the registry is not secured, add [literal]``--dest-tls-verify=false``.
+====
++
+. To start using the images from the internal registry please adapt the [literal]`NAMESPACE` value in file `/etc/sysconfig/uyuni-proxy-systemd-services.config`.
+
+[[air-gapped-solution-for-podman]]
+== Air-gapped solution for Podman
+
+This example illustrates deployment of containerized image on a machine with no access to internet.
```

Comment:
Not sure if you make a wording difference, but this procedure is for strict air-gapped setup where the Uyuni server and proxy network has no internet connection at all. This means the images need to be transferred using USB key.

Note that this doesn't prevent to have a mix of the two scenarios where the isolated network has an internal registry, but I suppose users with such setup will be smart enough to adapt the doc to their use case.